### PR TITLE
Add Signer

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: src/S3/S3Client.php [hunk 1]
```diff
@@ -21,6 +21,11 @@ protected function getServiceCode(): string
         return 's3';
     }
 
+    protected function getSignatureVersion(): string
+    {
+        return 's3';
```
Comment (Nyholm) [inline src/S3/S3Client.php]: v3?

Comment (jderusse) [reply src/S3/S3Client.php]: .. sadly.. no (This data is extracted from `API.json` file)

Comment (Nyholm) [reply src/S3/S3Client.php]: oh.

Comment (jderusse) [reply src/S3/S3Client.php]: For your Information, Every endpoints use V4, exept:
- `s3` thatis v4 + a custom header 
- `importexport` that use v2 (not implemented)

**Pull request merged**